DuckDuckGo/Subscription/ViewModel/SubscriptionFlowViewModel.swift

```diff
@@ -151,28 +154,63 @@ final class SubscriptionFlowViewModel: ObservableObject {
        
     }
     
+    // swiftlint:disable:next cyclomatic_complexity
     private func handleTransactionError(error: SubscriptionPagesUseSubscriptionFeature.UseSubscriptionError) {
+
+        var isStoreError = false
+        var isBackendError = false
+
         switch error {
-        
         case .purchaseFailed:
+            isStoreError = true
             transactionError = .purchaseFailed
         case .missingEntitlements:
+            isBackendError = true
             transactionError = .missingEntitlements
         case .failedToGetSubscriptionOptions:
+            isStoreError = true
             transactionError = .failedToGetSubscriptionOptions
         case .failedToSetSubscription:
+            isBackendError = true
             transactionError = .failedToSetSubscription
+        case .failedToRestoreFromEmail, .failedToRestoreFromEmailSubscriptionInactive:
+            isBackendError = true
+            transactionError = .generalError
         case .failedToRestorePastPurchase:
+            isStoreError = true
             transactionError = .failedToRestorePastPurchase
+        case .subscriptionNotFound:
+            isStoreError = true
+            transactionError = .generalError
         case .subscriptionExpired:
+            isStoreError = true
             transactionError = .subscriptionExpired
         case .hasActiveSubscription:
+            isStoreError = true
+            isBackendError = true
             transactionError = .hasActiveSubscription
         case .cancelledByUser:
+            isStoreError = true
             transactionError = nil
+        case .accountCreationFailed:
+            DailyPixel.fireDailyAndCount(pixel: .privacyProPurchaseFailureAccountNotCreated)
+            transactionError = .generalError
         default:
             transactionError = .generalError
         }
+
+        if isStoreError {
+            DailyPixel.fireDailyAndCount(pixel: .privacyProPurchaseFailureStoreError)
```

Comment:
Should these also be Daily pixels?

Reply:
Apparently yes:
Purchase Failure - Store Error
Type: Count Pixel + Daily Pixel
Trigger: When a subscription purchase fails because of a store error